pkgs/top-level/all-packages.nix

```diff
@@ -5585,6 +5585,8 @@ with pkgs;
 
   photon = callPackage ../tools/networking/photon { };
 
+  photofield = callPackage ../servers/photofield { };
```

Comment:
Please sort it alphabetically. This entry should not be after `photon`, please move it to the right place and I'll proceed further.

Reply:
The sorting is not consistent in many many places right now. It is next to p's which is usually good enough.